outline.md

```diff
@@ -0,0 +1,75 @@
+
+# Outline
+
+## **Session 1: Introduction to VEX and Programming**
+
+### Slides 1.1
+
+* Title slide: "Introduction to VEX Robotics Competition"
+* Overview of the VEX Robotics competition, including its rules, format, and what you will learn.
+* VEX is a global robotics competition for students, in which teams design build and program robots to compete against each other.
+* Matches are played on a square field, with teams competing in alliances of two and trying to earn points against oponents.
+* Present key components and hardware such as the VEX V5 Brain.
+* We can provide short video of past VEX competitions in action.
+
+### Discussion 1-1
+
+* What do you think the some of the features of the VEX V5 brain are? What kind of things can it allow our robot to do?
+
+### Demo 1-1
+
+* We will introduce the main hardware, the "VEX V5 Brain", and give physical demo of its use and features.
+
+### Slides 1-2
+
+* Title slide: Programming Languages and the Role of C
+
+* Introduction to the concept of programming languages.
+* Brief history of C and its relevance in modern technology. Provide examples of well known software written in C, such as the Linux kernel.
+* Comparisons between C and other languages, highlighting basics of performance, memory, and runtime efficiency.
+
+### Discussion 1-2
+
+* Why do you think C is still relevant today?
+* What type of software is C usually used for?
+
+### Example Code
+
+* "Hello World" program in C
```

Comment:
"Hello World in VEX"

Reply:
Change title layout likes above

Reply:
Hey Liang, I had to keep "Slides 1-2", "Discussion 1-2", etc. because Markdown does not let me have duplicate headings. I hope it's fine like this